packages/plugins/plugin-table/src/model/table-model.ts

```diff
@@ -86,7 +88,7 @@ export class TableModel extends Resource {
       return Object.fromEntries(
         fields.map((field, index: number) => {
           return [
-            getCellKey(index, 0),
+            fromCellKey({ col: index, row: 0 }),
```

Comment:
I find this name confusing, it doesn't indicate what's being created.

Reply:
ok to rename this but follow existing standards (see sheet) and make consistent (i.e., toX, fromX, etc.)

Reply:
Works for me. Agree get was suboptimal.

Reply:
A pattern I have used in the past is defining to and from as object properties. Then you can invoke them like:

`gridCellKey.fromCoordinates({ row, col })` or something like that.